libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html

```diff
@@ -3,6 +3,8 @@
     <div mainPage class="home-page">
       <elewa-group-elewa-hero></elewa-group-elewa-hero>
       <elewa-group-home-hero-section></elewa-group-home-hero-section>
+      <elewa-group-home-impact-section></elewa-group-home-impact-section>
+      <elewa-group-home-jobs-section></elewa-group-home-jobs-section>
```

Comment:
One of the jobs sections should be removed

Reply:
They are different component the jobs section being a componnt that was merged and mine being impact section

Reply:
Check the file please. There are two job sections. One shouldn't be there, check the design for the right placement.

Reply:
Sure thing. I've fixed it